src/views/camp/camp.scss

```diff
@@ -0,0 +1,80 @@
+@import "../../colors";
+@import "../../frameless";
+
+#view {
+    padding: 0;
+}
+.topImg {
+    margin-top: -10px;
+    padding: 0;
+    width: 100%;
+}
+.bubbles {
+    display: block;
+    margin: 0 auto;
+    margin-top: .938em;
+    width: 45%;
+}
+h3 {
+    border-bottom: 1px solid $ui-border;
+}
+.flex-row {
+    &.sidebar-row {
+        align-items: flex-start;
+
+        .body-copy {
+            width: $cols5;
+        }
+
+        .sidebar {
+            width: $cols5;
+        }
+    }
+}
+#particpate {
+    position: relative;
+    
+    h2, h3, p {
+        color: $ui-white;
+    }
+}
+#info {
+    h2, p {
+        color: $ui-white;
+    }
+}
+.info-content {
+    div {
+        max-width: $cols4;
+        text-align: center;
+    }
+}
+.infoImg {
+    transform: scale(.8);
+    -webkit-transform: scale(.8); /* Saf3.1+, Chrome */
+    -moz-transform: scale(.8); /* FF3.5+ */
+    -ms-transform: scale(.8); /* IE9 */
+    -o-transform: scale(.8); /* Opera 10.5+ */
+}
+.bottomImg {
```

Comment:
I did a little playing around with this in my browser (Chrome, Mac OSX) and I think I found it looked best with:
```css
.bottomImg {
    margin-bottom: -5px;
    padding: 0;
    margin-left: -4px;
    width: 102%;
}
```